common/djangoapps/student/tests/tests.py

```diff
@@ -281,7 +281,8 @@ def test_blocked_course_scenario(self, log_warning):
             course_id=self.course.id, is_valid=False
         )
         course_reg_code = shoppingcart.models.CourseRegistrationCode(code="abcde", course_id=self.course.id,
-                                                                     created_by=self.user, invoice=sale_invoice_1)
+                                                                     created_by=self.user, invoice=sale_invoice_1,
+                                                                     mode_slug='honor')
```

Comment:
nitpick: While you're here, could you update these lines to follow the indentation conventions?